src/schema/artist/index.js

```diff
@@ -155,18 +95,16 @@ export const ArtistType = new GraphQLObjectType({
           { rootValue: { articlesLoader } }
         ) => {
           const relayOptions = parseRelayOptions(args)
-
+          const gravityArgs = omit(args, ["first", "after", "last", "before"])
```

Comment:
I realized this last night...for auction results we don't want to include these Relay args when passing thru to the loader and underlying API.